packages/ui/DatePicker/index.tsx

```diff
@@ -1,18 +1,22 @@
 "use client";
 
-import { useState } from "react";
+import { useContext } from "react";
```

Comment:
changes to fix the clear icon in date picker inputs (1/6)

```suggestion
import { useContext, useState } from "react";
```